dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java

```diff
@@ -148,6 +153,84 @@ public void testGeneralDefaultPathMechanism() {
         final io.swagger.client.model.Workflow workflow = client.getWorkflow(invoke.getServiceID(), "");
     }
 
+    /**
+     * This tests endpoints that will be triggered by GitHub App webhooks.
+     * A service is created and a version is added for a release 1.0
+     */
+    @Test
+    public void testGitHubAppEndpoints() throws Exception {
+        CommonTestUtilities.cleanStatePrivate2(SUPPORT, false);
+        final ApiClient webClient = getWebClient("admin@admin.com");
+        WorkflowsApi client = new WorkflowsApi(webClient);
+
+        String serviceRepo = "DockstoreTestUser2/test-service";
+        String installationId = "1179416";
+
+        // Add service
+        io.swagger.client.model.Workflow service = client.addService(serviceRepo, "admin@admin.com", installationId);
+        assertNotNull(service);
+
+        // Add version
+        service = client.upsertServiceVersion(serviceRepo, "1.0", installationId);
+
+        assertNotNull(service);
+        assertEquals("Should have a new version", 1, service.getWorkflowVersions().size());
+        assertEquals("Should have 3 source files", 3, service.getWorkflowVersions().get(0).getSourceFiles().size());
+
+        // Add service as another user
+        service = client.addService(serviceRepo, "DockstoreTestUser2", installationId);
+        assertNotNull(service);
+        assertEquals("Should have 2 users, has " + service.getUsers().size(), service.getUsers().size(), 2);
```

Comment:
Flip the order of the last 2 parameters.  Only need a short message because it automatically prints out the expected and actual